content/WebAuthn/Concepts/Authenticator_Management/Implementation_Guidance/Deny_List.adoc

```diff
@@ -0,0 +1,127 @@
+== Deny list implementation guidance
+
+A deny list is a curated list of authenticators that should **not** be allowed to register and authenticate to your application. Any authenticator that is not part of your curated list should be allowed to register. This scenario works well for applications that want a degree of control to prevent vulnerable authenticators from being used in the application. The guidance given on this page will teach you how to implement a deny list in your application, that will limit certain authenticators from registering.
+
+This scenario works best when:
+
+* Your organization has a security organization actively monitoring for authenticator issues
+* You want to prevent users from registering authenticators like copyable passkeys
```

Comment:
Consider rewording:
* You want to prevent users from registering authenticators with certain properties, e.g. authenticators that can create copyable passkeys